tests/libs/libsquare/package.lisp

```diff
@@ -0,0 +1,2 @@
+(defpackage #:sbcl-librarian/tests/libs/libsquare
```

Comment:
This is totally minor, but I noticed that here you are using slashes in the package name, whereas below (in `tests/package.lisp`) you use dots. Any particular reason for this? One thing we have done for some of the quil projects is uses slashes in ASDF system names (for "secondary systems", this is needed for ASDF to find them) and dots in packages, but I couldn't really tell you why we do that.

Reply:
No good reason, I pretty much just cargo-culted [this package name](https://github.com/quil-lang/sbcl-librarian/blob/eb084c07d57ec5591b0e8feaa5256d5b38cda8c3/example/package.lisp#L1) from the example. I did try to copy the dot convention used in the Quilc packages for the test package, but I missed it here.

Now that you mention it, this package name should probably just be something like `sbcl-librarian.tests.libsquare`.

Reply:
Slashes in package names usually are used in "package inferred" ASDF systems. In such systems package names mirror a file structure and also are names of implicitly created secondary ASDF systems.